pandas/tests/test_graphics.py

```diff
@@ -1065,12 +1065,6 @@ def test_implicit_label(self):
         self._check_text_labels(ax.xaxis.get_label(), 'a')
 
     @slow
-    def test_explicit_label(self):
```

Comment:
Can you add this test back in, or say why you removed it? I'd guess people would rely on specifying label here implies a legend (even though it doesn't with Series.plot :frown: )